test/integration/etl.js

```diff
@@ -0,0 +1,45 @@
+const nock = require('nock');
+const fs = require('fs');
+
+const etl = require('../../app/lib/etl');
+const service = require('../../app/lib/syndicationService');
+
+function readFile(path) {
+  return fs.readFileSync(path, 'utf8');
+}
+
+const uri = `/all.xml?apikey=${process.env.SYNDICATION_API_KEY}&page=1`;
+
+function stubNoResults() {
```

Comment:
Can this function be combined with `stubOnePageOfResults` if `filePath` is added as a parameter?